src/client.ts

```diff
@@ -165,4 +183,4 @@ class Client {
   }
 }
 
-export { Client };
+export { Client, Publication };
```

Comment:
TS consumers will need to know what a publication is to provide the right handler.